OpenEdXMobile/src/main/java/org/edx/mobile/view/adapters/CourseOutlineAdapter.java

```diff
@@ -372,23 +376,26 @@ private void getRowViewForLeaf(ViewHolder viewHolder, final SectionRow row) {
                 viewHolder.lockedContent.setVisibility(View.VISIBLE);
             } else {
                 viewHolder.rowSubtitle.setText(R.string.not_available_on_mobile);
-                viewHolder.rowType.setIconColorResource(R.color.primaryXLightColor);
+                UiUtil.changeDrawableColor(context, viewHolder.rowTitle, R.color.primaryXLightColor);
             }
             viewHolder.rowSubtitlePanel.setVisibility(View.VISIBLE);
             viewHolder.rowSubtitle.setVisibility(View.VISIBLE);
         }
+        viewHolder.wholeSeparator.setVisibility(View.VISIBLE);
+        if (unit.isCompleted() || (isVideoMode && unit.isCompletedForVideos())) {
+            viewHolder.rowContainer.setBackgroundColor(ContextCompat.getColor(context, R.color.successXXLight));
+            viewHolder.wholeSeparator.setBackgroundColor(ContextCompat.getColor(context, R.color.successXLight));
+            viewHolder.rowCompleted.setImageDrawable(ContextCompat.getDrawable(context, R.drawable.ic_green_check));
+            viewHolder.rowCompleted.setVisibility(View.VISIBLE);
+        }
     }
 
     private void checkAccessStatus(boolean isDenialFeatureBasedEnrolments, final ViewHolder viewHolder, final CourseComponent unit) {
         if (!isDenialFeatureBasedEnrolments) {
             dbStore.isUnitAccessed(new DataCallback<Boolean>(true) {
                 @Override
                 public void onResult(Boolean accessed) {
-                    if (accessed) {
-                        viewHolder.rowType.setIconColorResource(R.color.primaryXLightColor);
-                    } else {
-                        viewHolder.rowType.setIconColorResource(R.color.primaryBaseColor);
-                    }
+                    UiUtil.changeDrawableColor(context, viewHolder.rowTitle, accessed ? R.color.primaryXLightColor : R.color.primaryBaseColor);
```

Comment:
no need to this method, cuz the app is not showing access status from now.

Reply:
I'm not sure about this as it is currently highlighting the block which are not available on mobile

Reply:
@saeedbashir @mumer92  can you please confirm.